__test__/Checkbox.test.js

```diff
@@ -0,0 +1,125 @@
+const isChecked = (elementHandle) => page.evaluate((element) => {
+    const innerCheckbox = element.shadowRoot.querySelector("input");
+    return innerCheckbox.checked;
+}, elementHandle);
+
+describe('Checkbox', () => {
+    const getPageURL = (size, label, labelPosition, checkState, disabled) => 
+        `http://localhost:6006/iframe.html?viewMode=story&id=components-checkbox--small&args=size:${size};label:${label};labelPosition:${labelPosition};checkState:${checkState};${disabled?"disabled:!true":""}`;
+    // vision test
+    describe.each(["small", "large"])("%s", (size) => {
+        describe.each(["left", "right"])("%s", (labelPosition) => {
+            describe.each(["unchecked", "indeterminate", "checked"])("%s", (checkState) => {
+                describe.each([false, true])("%s", (disabled) => {
+                    const baseURL = getPageURL(size, "test label", labelPosition, checkState, disabled);
+
+                    it('enable', async () => {
```

Comment:
Inconsistent when disabled state

```suggestion
                    it('normal', async () => {
```